### PR TITLE
block for ready chunks

### PR DESCRIPTION
**Pull request by mental32**

### CODE REVIEW

File: src/database/src/lib.rs [hunk 1]
```diff
@@ -63,21 +63,21 @@ cfg_if! {
...
-        } else {
-            let _ = diesel::sql_query("PRAGMA journal_mode=WAL").execute(conn)?;
-            let _ = diesel::sql_query("PRAGMA synchronous=NORMAL").execute(conn)?;
-            let _ = diesel::sql_query("PRAGMA busy_timeout=50000").execute(conn)?;
-            let _ = diesel::sql_query("PRAGMA foreign_keys = ON").execute(conn)?;
-        }
-    }
-    Ok(())
-}
+// fn create_database(conn: &crate::DbConnection) -> Result<(), diesel::result::Error> {
```
Comment (vgarleanu) [inline src/database/src/lib.rs:66]: why is this commented out?

File: src/database/src/lib.rs [hunk 2]
```diff
@@ -119,7 +119,7 @@ pub fn get_conn_devel() -> Result<crate::DbConnection, diesel::result::Connectio
                 "postgres://postgres:dimpostgres@postgres/dim_devel",
             )?;
         } else {
-            let mut conn = DbConnection::establish("./dim_devel.db;foreign_keys=true;")?;
+            let conn = DbConnection::establish("./dim_devel.db;foreign_keys=true;")?;
```
Comment (vgarleanu) [inline src/database/src/lib.rs:122]: can you remove the foreign_keys=true thingie including the `;`

File: src/database/src/lib.rs (hunk 1 above)
Comment (mental32) [reply src/database/src/lib.rs:66]:
```
warning: unused variable: `log`
   --> src/database/src/lib.rs:155:5
    |
155 |     log: Option<&Logger>,
    |     ^^^ help: if this is intentional, prefix it with an underscore: `_log`
    |
    = note: `#[warn(unused_variables)]` on by default

warning: variable does not need to be mutable
   --> src/database/src/lib.rs:122:17
    |
122 |             let mut conn = DbConnection::establish("./dim_devel.db;foreign_keys=true;")?;
    |                 ----^^^^
    |                 |
    |                 help: remove this `mut`
    |
    = note: `#[warn(unused_mut)]` on by default

warning: variable does not need to be mutable
   --> src/database/src/lib.rs:165:17
    |
165 |             let mut conn = DbConnection::establish("./dim.db")?;
    |                 ----^^^^
    |                 |
    |                 help: remove this `mut`

warning: function is never used: `create_database`
  --> src/database/src/lib.rs:66:4
   |
66 | fn create_database(conn: &crate::DbConnection) -> Result<(), diesel::result::Error> {
   |    ^^^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default

warning: function is never used: `internal_get_conn_custom`
   --> src/database/src/lib.rs:175:4
    |
175 | fn internal_get_conn_custom(
    |    ^^^^^^^^^^^^^^^^^^^^^^^^

warning: 5 warnings emitted
```

File: src/database/src/lib.rs (hunk 2 above)
Comment (mental32) [reply src/database/src/lib.rs:122]:
```suggestion
            let conn = DbConnection::establish("./dim_devel.db;")?;
```

**Pull request merged**